### ISSUE TITLE
Binary releases

### ISSUE THREAD

**Issue by marnovo** [vitor251093/wineskin#35]
Do you plan to provide binary releases (at least for every minor non-beta release)?

**Comment by Gcenx** [comment]
I would assume that will be the case when its decided that the project is stable enough to warrant a release.

If you can't wait you could always download and compile with Xcode or just download my winery.app from my [WineskinServer repot](https://github.com/Gcenx/WineskinServer)

That will download my current Development version of Wineskin, master wrapper and will have select engines available for direct download (Github 1GB size limit).

**Comment by marnovo** [comment]
Sure, I have compiled it without issues. The question was more aimed at the less skilled from the community, for the future indeed. Thanks.

**Comment by Gcenx** [comment]
> Sure, I have compiled it without issues. The question was more aimed at the less skilled from the community, for the future indeed. Thanks.

I figured that was the case, the link is aimed at the the type who can't compile/put together the final product.

Eventually this project will take over, Im not sure on that time frame that's more between doh123 and @vitor251093

**Comment by vitor251093** [comment]
Once we have done the most urgent fixes and updates we can release one :)

I guess the point is just which of the actual open issues are a must for the first release.

In my opinion, those are the legacy support and the post-Mojave branches. @Gcenx @chrisballinger  there is any other change/enhancement which you consider a must?

**Comment by chrisballinger** [comment]
Nah, I think just having some sort of baseline "release" will be good. Also I'd lean towards dropping all legacy support.

**Comment by Gcenx** [comment]
@vitor251093 let me finish remaking my none mergeable branch so we can have WineskinX11 dropped then and the XQuartz issue in the other thread then I think we should be good

**Comment by clobber** [comment]
Thanks all for taking this on - glad the future of Wineskin now has a home on GitHub with active contributors. I know @MaddTheSane is very good at reviving old macOS projects that have fallen to code rot. Just mentioning him here in case this interests him.
